### PR TITLE
Topic0 2

### PR DESCRIPTION
**Pull request by AnaCDomingo**
Here are the excercises of Topic 0 - Ex 2.

### CODE REVIEW

**Comment by AnaCDomingo** [comment]
I've finished topic 0!

File: 00-HTML-CSS-basics/Ex01_V_VI.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	</head>
+
+	<body>
+		<header>
+			<h1> Excercises V and VI </h1> 
+		</header>
+		<img src="http://orig00.deviantart.net/bb48/f/2014/088/8/9/jack_frost_persona__by_flaremor-d7c4r75.png" alt="This is Jack Frost from the SMT Series" width="20%"> <br>
+		<p> This is Jack Frost from the SMT Series <br> </p>
+		<video width="50%" preload="none" controls>
+			<source src="Media/Jack_Frost_Converted.mp4" type="video/mp4">
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex01_V_VI.html:16]: We use to add an extra message for the case when the browser does not support a tag. Not need to apply it.

File: 00-HTML-CSS-basics/Ex02/css/style.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.sidebar {
+	font-size: 25px;
+	display: inline-block;
+	position: relative;
+	height: 100%;
+	border-radius: 10px;
+	float: right;
+}
+div.midsect {
+	display: relative;
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex02/css/style.css:67]: I think you want to use, `position:relative`

File: 00-HTML-CSS-basics/Ex02/css/style.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	margin: auto 2%;
+	height: 600px;
+	float: left;
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex02/css/style.css:70]: You can avoid the `float: left`, with `position: relative` should be enough

File: 00-HTML-CSS-basics/Ex02/css/style.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+}
+.content footer {
+	display: block;
+	position: absolute;
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex02/css/style.css:74]: Do not use `display:block` with `position: absolute`. All elements that are absolute, are automatically treated as display:block. Also, I recommend you to remove the float with clear fix here.

File: 00-HTML-CSS-basics/Ex02/css/style.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+	bottom: 0;
+	width: 100%;
+	margin: 2% auto; 
+}
+.content article {
+	padding: 1% 5%;
+}
+/* [class$='r'] {
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex02/css/style.css:82]: Try to not leave comments in the code

File: 00-HTML-CSS-basics/Ex02/css/layout_styles.css [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+	display: inline-block;
+	position: relative;
+	width: 33.3%;
+	margin: 2% 2%;
+	z-index: 0;
+	height: auto;
+}
+div.inner_box {
+	opacity: 0;
+	display: inline-block;
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex02/css/layout_styles.css:26]: What happens if you remove the `display: inline-block` from inner_box?

File: 00-HTML-CSS-basics/Ex02/css/layout_styles.css [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 90%;
+	height: 90%;
+	z-index: 1;
+	padding: 5%;
+	background-color: black;
+	color: white;
+	text-align: center;
```
Comment (mtulian) [inline 00-HTML-CSS-basics/Ex02/css/layout_styles.css:36]: The `text-align: center` is already set in the `html` tag. Try to avoid additional css

**Review by mtulian — COMMENTED** [review]
Let's address the last comments and close the topic 0

File: 00-HTML-CSS-basics/Ex02/css/style.css (hunk 3 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/style.css:70]: :+1:

File: 00-HTML-CSS-basics/Ex01_V_VI.html (hunk 1 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex01_V_VI.html:16]: I'll take it in consideration.

File: 00-HTML-CSS-basics/Ex02/css/style.css (hunk 2 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/style.css:67]: Yes, I've got messed up e.e

File: 00-HTML-CSS-basics/Ex02/css/style.css (hunk 3 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/style.css:70]: Good.

File: 00-HTML-CSS-basics/Ex02/css/style.css (hunk 5 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/style.css:82]: That was for the previous excercise.

File: 00-HTML-CSS-basics/Ex02/css/style.css (hunk 4 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/style.css:74]: Thank you, I'll have it in mind.

File: 00-HTML-CSS-basics/Ex02/css/layout_styles.css (hunk 7 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/layout_styles.css:36]: :+1:

File: 00-HTML-CSS-basics/Ex02/css/layout_styles.css (hunk 6 above)
Comment (AnaCDomingo) [reply 00-HTML-CSS-basics/Ex02/css/layout_styles.css:26]: I think that nothing will happen, because it's inside another div.

**Review by mtulian — APPROVED** [review]

**Pull request merged**